rtrlib/transport/tcp/tcp_transport.h

```diff
@@ -31,10 +31,17 @@
  */
 struct tr_tcp_config {
 	char *host;
+	char *vrfname;
 	char *port;
 	char *bindaddr;
 };
 
+struct tr_tcp_socket {
```

Comment:
This should definitely not be in the public header. Maybe in the private header but even that should not be necessary.